src/js/steps/letters/init.js

```diff
@@ -41,7 +41,7 @@ module.exports = function ($events, options) {
       }
     });
 
-    $letters.on('click', '.letter', function () {
+    $letters.on('click', '.letter:not(.nonclickable)', function () {
```

Comment:
Special characters are at the start/finish of the name, they link to the start/finish of the book. The nonclickable part is only for spaces and hyphens.